ccdproc/image_collection.py

```diff
@@ -112,6 +112,9 @@ def __init__(self, location=None, keywords=None, info_file=None,
         if keywords:
             self.keywords = keywords
 
+        if extension:
```

Comment:
This should change to `if extension is not None` because `0` is a valid extension number but will evaluate to `False`.